scripts/config.py

```diff
@@ -68,9 +67,7 @@ def prompt(message, default=None):
         return raw_input( '%s: ' % (message) )
 
 def random_string(len):
-    chars = string.ascii_uppercase + string.digits + string.ascii_lowercase
-    arr = [random.choice(chars) for i in range(len)]
-    return ''.join(arr)
+    return urandom(len).encode('base-64').rstrip()
```

Comment:
The problem with base-64 is that the strings end with "=" or "==" much of the time.

I looked into this a bit. The way to do it is the way you did it the first time, but put this line at the top of the function:

random = random.SystemRandom()

it uses urandom under the hood. see here: https://docs.python.org/3/library/random.html